docs/connector-development/README.md

```diff
@@ -54,15 +54,15 @@ and choose the relevant template. This will generate a new connector in the `air
 
 Search the generated directory for "TODO"s and follow them to implement your connector. For more detailed walkthroughs and instructions, follow the relevant tutorial:
 
-* [Building a Python source ](../tutorials/building-a-python-source.md) 
-* [Building a Python destination](../tutorials/building-a-python-destination.md)
-* [Building a Java destination ](../tutorials/building-a-java-destination.md)
+* [Building a Python source ](tutorials/building-a-python-source.md) 
```

Comment:
@sherifnada shouldn't we also have smth about the CDK here?

Reply:
There's a note about the CDK at the top of the document, but I can definitely reiterate here. I'll let @sherifnada respond.

Reply:
can't hurt to add a line like `most of these tutorials demonstrate how to use the [CDK](LINK) to build a connector`

Reply:
Added two links to the CDK tutorials!